docs/getting-started/installation.rst

```diff
@@ -62,8 +63,8 @@ using the following commands from within a terminal:
     cd pymedphys
 
     conda config --add channels conda-forge
-    conda create --name pmp python=3.7 # Optional.
-    conda activate pmp # Optional
+    conda create --name pmp python=3.7
+    conda activate pmp
     conda install pymedphys --only-deps
```

Comment:
Maybe for "bleeding edge" we should link users to the contributor docs. Skipped to the part where the installation happens.

Reply:
Hmmmm ... I don't know if that will work..

Reply:
Also makes sense, I think.